transformato/mutate.py

```diff
@@ -120,13 +124,16 @@ def perform_mutations(
     # Turn off hydrogens
 
     if mutation_list["hydrogen-lj"]:
-        print("####################")
-        print(f"Hydrogen vdW scaling in step: {i.current_step} with lamb: {0.0}")
-        print("####################")
-        i.write_state(
-            mutation_conf=mutation_list["hydrogen-lj"],
-            lambda_value_vdw=0.0,
-        )
+        for lambda_value in np.linspace(
+            0.75, 0, nr_of_mutation_steps_lj_of_hydrogens + 1
```

Comment:
`np.linspace(1,0,1)` returns `1` instead of `0`, which should be the default parameter.
I think this is best solved by an `if ... else` clause testing if `nr_of_mutation_steps_lj_of_hydrogens==1` (see previous comment)

Reply:
Or we just revert it by using `np.linspace(0,0.75, 5)[::-1]`, then `1` would give us as desired `0`.

Reply:
yes, that's also a possibility! but can we use `reversed` instead of `[::-1]`.